web_src/css/repo.css

```diff
@@ -2759,6 +2759,7 @@ tbody.commit-list {
   border: none;
   user-select: none;
   color: inherit;
+  display: flex !important;
```

Comment:
Please document why the `!important` is necessary.
Otherwise, we will never know if it is still needed.

Reply:
to be honest: i just copied the line from gt-df. Will need to check if the important is really neccessary.

Reply:
just checked - it's not needed. so i removed it.